### PR TITLE
Feature/derp 223 view accordion component

### PR DESCRIPTION
**Pull request by philwolstenholme**

### CODE REVIEW

File: config/install/config_split.config_split.development.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ label: Development
 description: 'Development related modules and configuration.'
 folder: ../config/splits/development
 module:
+  config_update: 0
```
Comment (philwolstenholme) [inline config/install/config_split.config_split.development.yml:9]: We need these enabled locally for Luke's script/Lando tooling for copying profile config back into the site. The report it uses relies on both https://www.drupal.org/project/config_update and the submodule `config_update_ui` being enabled.

**Comment by philwolstenholme** [comment]
@joz-cti I removed some Claro block config from the profile as it's the same config as Claro itself provides when Claro is installed, so I don't _think_ it needs to be part of the profile? I'll do a full site install from the profile to make sure everything still works and I'll update the PR if there's an issue, and update this comment if there isn't an issue.

**Comment by joz-cti** [comment]
> @joz-cti I removed some Claro block config from the profile as it's the same config as Claro itself provides when Claro is installed, so I don't _think_ it needs to be part of the profile? I'll do a full site install from the profile to make sure everything still works and I'll update the PR if there's an issue, and update this comment if there isn't an issue.

I wasn't sure whether I should add contrib module config into that config folder or not.
I think it might have been required if we were running `drush si --existing-config`? Might one one to test.

**Comment by philwolstenholme** [comment]
> drush si --existing-config

I think `drush si --existing-config` would be fine, because the `config/sync` config at a site level will have all the config exported from contrib/core modules. When we run `drush cex -y` _everything_ gets exported to `config/sync`, regardless of whether it's inside the profile or not.

**Comment by philwolstenholme** [comment]
Let's not merge this yet as I'm getting this error when doing a `lando drush si -y gla_core_profile --site-name='alpha-test-site-1' --account-name=root --account-pass=password install_configure_form.enable_update_status_emails=NULL --verbose
`

```
 [error]  TypeError: Drupal\Core\Field\FormatterBase::__construct(): Argument #4 ($settings) must be of type array, null given, called in /app/web/core/lib/Drupal/Core/Field/FormatterBase.php on line 78 in Drupal\Core\Field\FormatterBase->__construct() (line 64 of /app/web/core/lib/Drupal/Core/Field/FormatterBase.php) #0 /app/web/core/lib/Drupal/Core/Field/FormatterBase.php(78): Drupal\Core\Field\FormatterBase->__construct('url_embed', Array, Object(Drupal\field\Entity\FieldConfig), NULL, 'hidden', 'default', Array)
#1 /app/web/core/lib/Drupal/Core/Field/FormatterPluginManager.php(64): Drupal\Core\Field\FormatterBase::create(Object(Drupal\Core\DependencyInjection\ContainerBuilder), Array, 'url_embed', Array)
```

Probably unrelated but I'll try and fix it.

**Comment by joz-cti** [comment]
>  Argument #4 ($settings) must be of type array

Have we created a custom formatter or using one incorrectly in custom code?
The quickest way to find this would be `git bisect` to narrow down to the commit it was introduced.

**Review by joz-cti — APPROVED** [review]

**Pull request merged**